doc/man3/EVP_PKEY_meth_new.pod

```diff
@@ -350,6 +352,9 @@ then the built-in objects.
 
 EVP_PKEY_meth_add0() adds B<pmeth> to the user defined stack of methods.
 
+EVP_PKEY_meth_remove() removes an B<EVP_PKEY_METHOD> object added by
+EVP_PKEY_meth_new().
```

Comment:
Oops, I think you meant `EVP_PKEY_meth_add0`

Reply:
Sorry that I didn't catch this earlier :-/

Reply:
Ahh, originally I wanted to express 'created by `_new` and added by `_add0`'. Now seems just mention `_add0` is enough...